website/docs/advanced/instantiate_objects.md

```diff
@@ -44,66 +44,58 @@ class ObjectConf(Dict[str, Any]):
 models.py
 ```python
 class Foo:
-  def __init__(self, x: int, y: int) -> None:
+
+  # target: models.Foo
+  def __init__(self, x: int, y: int, z:int = 3) -> None:
     self.x = x
     self.y = y
+    self.z = z
 
+  # target: models.Foo.class_method
   @classmethod
-  def class_method(self, z: int) -> Any:
-    return self(z, 10)
-
+  def class_method(self, x: int) -> Any:
+    return self(x, 10)
+    
+  # target: models.Foo.static_method
   @staticmethod
   def static_method(z: int) -> int:
     return z + 1
 
+# target: models.bar
 def bar(z: int) -> int:
   return z + 2
+
 ```
+
+For example, to instantiate a models.Foo object:
+
 config.yaml
 ```yaml
 myobject:
```

Comment:
rename myobject to foo, then instantiate becomes:

foo : Foo = instantiate(cfg.foo)